.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
+*.pyo
```

Comment:
Common thing to do here is [`*.py[cod]`]( https://github.com/github/gitignore/blob/1fb9ea045e619c61e223371533cdea118efbea5d/Python.gitignore#L3 ). Also would be good to add `__pycache__/` while we are at it.